python/ingestion/gcs_to_bq_util.py

```diff
@@ -7,6 +7,9 @@
 from google.cloud import bigquery, storage
 
 
+DATA_DIR = os.path.join(os.sep, 'app', 'data')
```

Comment:
will it always be `app` regardless of where this is running (docker locally, gcp, etc) ? That's what I wasn't sure

Reply:
This code will only run in the docker environment